CHANGELOG.md

```diff
@@ -21,6 +20,13 @@ _None_
 
 _None_
 
+## 11.0.0
+
+### Breaking Changes
+
+- Make `ios_check_beta_deps` use the `Podfile.lock` instead of `Podfile` for its detection, and also be able to detect Pods referenced by commits and branches.  
+   If your `Fastfile` called this action with an explicit `podfile: …` argument, you'll have to update the call to use `lockfile:` instead (or rely on defaults). [#557]
```

Comment:
I didn't notice the whitespace in the second line in the PR before. Fixing it here.

```suggestion
- Make `ios_check_beta_deps` use the `Podfile.lock` instead of `Podfile` for its detection, and also be able to detect Pods referenced by commits and branches.
  If your `Fastfile` called this action with an explicit `podfile: …` argument, you'll have to update the call to use `lockfile:` instead (or rely on defaults). [#557]
```